### PR TITLE
Retag Heptio EventRouter

### PR DESCRIPTION
**Pull request by omegas27**
_Retagger_

Needed for https://github.com/giantswarm/eventrouter-app/tree/add-eventrouter/helm/eventrouter-app as the image does not define a user

### CODE REVIEW

File: images.yaml [hunk 1]
```diff
@@ -338,6 +338,15 @@
   tags:
   - sha: 97697df680ea4c7e70c4cb4af5bdc44f7a70b25be7afde70bd921a658e4c62ec
     tag: v0.30.1
+- name: gcr.io/heptio-images/eventrouter
```
Comment (MarcelMue) [inline images.yaml]: argh sorting needs to be alphabetical

Comment (omegas27) [reply images.yaml]: Do I sort by repo or image name?

Comment (MarcelMue) [reply images.yaml]: Repo then image - so it should be up there with the other `gcr.io` ones

Comment (omegas27) [reply images.yaml]: Thanks :) It should be better now :)

**Pull request merged**